specs/TrackingPrevention.md

```diff
@@ -0,0 +1,205 @@
+
+For reference, in the screenshot below, this API sets the levels of tracking prevention as a WebView2 API.
+
+![Edge Tracking Prevention](images/TrackingPreventionLevels.png)
+
+# Examples
+## IsTrackingPreventionEnabled
+
+```c#
+/// Create WebView Environment with option that disable tracking prevention feature.
+
+void CreateEnvironmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    // If we're displaying app content or otherwise are not worried about tracking, we can
+    // disable the tracking prevention feature to improve runtime performance.
+    options.IsTrackingPreventionEnabled = false;
+    CoreWebView2Environment environment = await CoreWebView2Environment.CreateAsync(BrowserExecutableFolder, UserDataFolder, options);
+}
+```
+
+```cpp
+Microsoft::WRL::ComPtr<ICoreWebView2EnvironmentOptions3> options3;
+if (options.As(&options3) == S_OK)
+{
+    // If we're displaying app content or otherwise are not worried about tracking, we can
+    // disable the tracking prevention feature to improve runtime performance.
+    CHECK_FAILURE(options3->put_IsTrackingPreventionEnabled(FALSE));
+}
+```
+
+## TrackingPreventionUserPreference
+/// Example to set level of tracking preference.
+
+```c#
+/// Below example set level of tracking preference.
+
+void SetTrackingPreventionLevel(CoreWebView2TrackingPreventionLevel value)
+{
+    WebViewProfile.TrackingPreventionLevel = value;
+    MessageBox.Show(this, "Tracking prevention level is set successfully", "Tracking Prevention Level");
+}
+```
+
+```cpp
+void SettingsComponent::SetTrackingPreventionLevel(
+    COREWEBVIEW2_TRACKING_PREVENTION_LEVEL value)
+{
+    wil::com_ptr<ICoreWebView2_13> webView2_13;
+    webView2_13 = m_webView.try_query<ICoreWebView2_13>();
+
+    if (webView2_13)
+    {
+        wil::com_ptr<ICoreWebView2Profile> profile;
+        CHECK_FAILURE(webView2_13->get_Profile(&profile));
+
+        auto profile5 = profile.try_query<ICoreWebView2Profile5>();
+        if (profile5)
+        {
+            CHECK_FAILURE(profile5->put_TrackingPreventionLevel(value));
+            MessageBox(
+                nullptr,
+                L"Tracking prevention level is set successfully",
+                L"Tracking Prevention Level", MB_OK);
+        }
+    }
+}
+```
+
+# API Details
+```
+/// Tracking prevention levels.
+[v1_enum] typedef enum COREWEBVIEW2_TRACKING_PREVENTION_LEVEL {
+  /// Tracking prevention is turned off.
+  COREWEBVIEW2_TRACKING_PREVENTION_LEVEL_OFF,
+  /// The least restrictive level of tracking prevention. Set to this level to
+  /// protect against malicious trackers but allows most other trackers and personalize content and ads.
+  ///
+  /// See [table](/microsoft-edge/web-platform/tracking-prevention#current-tracking-prevention-behavior)
+  /// for finegrain information on what is being blocked/unblocked with this level. This can change with different Edge versions.
```

Comment:
Here and elsewhere it should be fine-grained

```suggestion
  /// for fine-grained information on what is being blocked with this level. This can change with different Edge versions.
```